bodhi/server/consumers/masher.py

```diff
@@ -1240,6 +1240,40 @@ def _wait_for_pungi(self, pungi_process):
         self.log.debug('Path: %s', self.path)
         self._checkpoints['completed_repo'] = self.path
 
+    def _wait_for_repo_signature(self):
+        """Wait for a repo signature to appear."""
+        notifications.publish(
+            topic="repo.signature.wait",
+            msg=dict(repo=self.id, agent=self.agent, path=self.path),
+            force=True,
+        )
```

Comment:
Shouldn't we only send this message if we are going to wait (i.e., do this inside the if block below)?